spec/requests/workers_spec.rb

```diff
@@ -3,8 +3,32 @@
 RSpec.describe WorkersController do
   describe "worker controller" do
     HEADERS = { "ACCEPT" => "application/json" }
-    let(:worker) {create(:worker)}
-    let(:ticket) {create(:ticket, worker:worker)}
+
+    before do
+      post '/users', :params => { "user": {
```

Comment:
do we need these requests ? can we do it on db level?

Reply:
let user don't work